collectors/deployment.go

```diff
@@ -83,6 +83,12 @@ var (
 		[]string{"namespace", "deployment"}, nil,
 	)
 
+	descDeploymentStrategyRollingUpdateMaxSurge = prometheus.NewDesc(
```

Comment:
@hangyan You should also add `ch <- descDeploymentStrategyRollingUpdateMaxSurge` to `Describe` function to tell prometheus client go scrape the max surge metrics.

Others lgtm